readme.html

```diff
@@ -13,4 +13,19 @@
 and limitations under the License.
 -->
 
-<p>Elasticsearch installations can be configured to allow REST API access without any type of authentication. The app therefore marks <b>username</b> and <b>password</b> as optional parameters. If specified, the app will end up using these for generating the basic authentication header used in the various Elasticsearch REST endpoints.<br>Connection can be configured over HTTP or HTTPS, so if <b>test connectivity</b> fails please check the protocol.</p>
+<p>Elasticsearch installations can be configured to allow REST API access without any type of authentication.
+    The app therefore marks <b>username</b> and <b>password</b> as optional parameters. If specified, the app
+    will end up using these for generating the basic authentication header used in the various Elasticsearch
+    REST endpoints.<br>Connection can be configured over HTTP or HTTPS, so if <b>test connectivity</b>
```

Comment:
REST endpoints.<br>The connection can be configured over HTTP or HTTPS, so if <b>test connectivity</b>